docs/tutorials/conversational_search/conversational_search_with_Bedrock_Claude.md

```diff
@@ -0,0 +1,197 @@
+        "service_name": "bedrock",
+        "model": "anthropic.claude-v2"
+    },
+    "actions": [
+        {
+            "action_type": "predict",
+            "method": "POST",
+            "headers": {
+                "content-type": "application/json"
+            },
+            "url": "https://bedrock-runtime.${parameters.region}.amazonaws.com/model/${parameters.model}/invoke",
+            "request_body": "{\"prompt\":\"\\n\\nHuman: ${parameters.inputs}\\n\\nAssistant:\",\"max_tokens_to_sample\":300,\"temperature\":0.5,\"top_k\":250,\"top_p\":1,\"stop_sequences\":[\"\\\\n\\\\nHuman:\"]}"
+        }
+    ]
+}
+```
+
+Note the connector ID; you will use it to create the model.
+
+2. Create model:
+```
+POST /_plugins/_ml/models/_register?deploy=true
+{
+    "name": "Bedrock Claude2 model",
+    "function_name": "remote",
+    "description": "Bedrock Claude2 model",
+    "connector_id": "your_connector_id"
+}
+```
+
+Note the model ID; you will use it in the following steps.
+
+3. Test the model:
+```
+POST /_plugins/_ml/models/your_model_id/_predict
+{
+    "parameters": {
+      "inputs": "Who won the world series in 2020?"
+    }
+}
+```
+Sample response:
+```
+{
+  "inference_results": [
+    {
+      "output": [
+        {
+          "name": "response",
+          "dataAsMap": {
+            "type": "completion",
+            "completion": " The Los Angeles Dodgers won the 2020 World Series, defeating the Tampa Bay Rays 4 games to 2. The World Series was played at a neutral site in Arlington, Texas due to the COVID-19 pandemic. It was the Dodgers' first World Series championship since 1988.",
+            "stop_reason": "stop_sequence",
+            "stop": "\n\nHuman:"
+          }
+        }
+      ],
+      "status_code": 200
+    }
+  ]
+}
+```
+
+## 2. Conversational search
+
+### 2.1 Create pipeline
+```
+PUT /_search/pipeline/my-conversation-search-pipeline-claude2
+{
+  "response_processors": [
+    {
+      "retrieval_augmented_generation": {
+        "tag": "Demo pipeline",
+        "description": "Demo pipeline Using Bedrock Claude2",
+        "model_id": "your_model_id_created_in_step1",
+        "context_field_list": [
+          "text"
+        ],
+        "system_prompt": "You are a helpful assistant",
+        "user_instructions": "Generate a concise and informative answer in less than 100 words for the given question"
+      }
+    }
+  ]
+}
+
+```
+
+### 2.2 Search
+Conversational search has some extra parameters you specify in `generative_qa_parameters`:
+```
+GET /qa_demo/_search?search_pipeline=my-conversation-search-pipeline-claude2
+{
+  "query": {
+    "match": {
+      "text": "What's the population increase of New York City from 2021 to 2023?"
+    }
+  },
+  "size": 1,
+  "_source": [
+    "text"
+  ],
+  "ext": {
+    "generative_qa_parameters": {
+      "llm_model": "bedrock/claude",
+      "llm_question": "What's the population increase of New York City from 2021 to 2023?",
```

Comment:
Since this is a conversational search, can you try a follow up questions to demo this is a on-going conversation? Maybe we can try another questions like "based on the population increase in New York, is it the fastest growing city in the world?", or other follow up questions.

Reply:
make sense, will add a following up question